open_humans/account_views.py

```diff
@@ -44,6 +44,8 @@ def post(self, request, *args, **kwargs):
         django's HttpResponseRedirect, which doesn't quite handle it correctly.
         """
         ret = super().post(self, request, *args, **kwargs)
+        if ret.url == 'None':
```

Comment:
There's an `AttributeError` below. I had thought it was to catch a missing `url` attribute. Is that not the case?